### PR TITLE
Prism: Changes to enable OGS and UKM for use with Prism

### PR DESCRIPTION
**Pull request by trflynn89**
This enables OGS and UKM at startup, and sets their respective URLs, to communicate with a local Prism server.

### CODE REVIEW

File: components/ukm/ukm_reporting_service.cc [hunk 1]
```diff
@@ -25,6 +25,15 @@
 #include "components/ukm/ios/ukm_reporting_ios_util.h"
 #endif
 
+#include "rebel/chrome/browser/prism/prism_buildflags.h"  // Needed for PRISM_ENABLED.
+#if BUILDFLAG(PRISM_ENABLED)
+// FIXME: This is to allow overriding the UKM server URL via command line. Can
+//        we set UKM's URL from //rebel without changing this file somehow?
+#ifdef NDEBUG
+#undef NDEBUG
```
Comment (debinzhang3) [inline components/ukm/ukm_reporting_service.cc:33]: does this change have a local impact only? I

Comment (trflynn89) [reply components/ukm/ukm_reporting_service.cc:33]: Yes, local only, and has no impact on the optimizations used to compile this file.

File: components/ukm/ukm_reporting_service.cc [hunk 2]
```diff
@@ -25,6 +25,15 @@
 #include "components/ukm/ios/ukm_reporting_ios_util.h"
 #endif
 
+#include "rebel/components/ukm/prism_buildflags.h"  // Needed for PRISM_ENABLED.
```
Comment (debinzhang3) [inline components/ukm/ukm_reporting_service.cc:28]: is this prism_buildflags.h file included in this PR? I cannot find it. or was it in the branch already?

Comment (trflynn89) [reply components/ukm/ukm_reporting_service.cc:28]: It gets generated at build time, see: https://github.com/RebelBrowser/rebel/blob/5cc1596e4f19efa9bd8d432d56e78dfff6d77d5f/rebel/components/ukm/BUILD.gn

**Review by debinzhang3 — APPROVED** [review]

**Pull request merged**